.github/workflows/settings_checks.yml

```diff
@@ -181,7 +181,7 @@ jobs:
         run: exit 1
 
       - name: Respond to PR if no invalid parameters are found
-        if: env.INVALID_PARAMETERS == '' && env.PARAM_STORE_VARS == 'true'
```

Comment:
See https://github.com/department-of-veterans-affairs/vets-api/pull/24846#discussion_r2473544531 for why we still want to check if `env.PARAM_STORE_VARS == 'true`